arch/arm64/configs/sun50iw1p1smp_linux_defconfig

```diff
@@ -118,7 +118,7 @@ CONFIG_CFS_BANDWIDTH=y
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_BLK_CGROUP=y
 # CONFIG_DEBUG_BLK_CGROUP is not set
-CONFIG_CHECKPOINT_RESTORE=y
+# CONFIG_CHECKPOINT_RESTORE is not set
```

Comment:
this parameter is valid in debug builds only. Since the later changes disable the debug build this parameter is not set anymore.